test/images/mount-tester/mt.go

```diff
@@ -51,6 +52,7 @@ func init() {
 	flag.StringVar(&readFileContentPath, "file_content", "", "Path to read the file content from")
 	flag.StringVar(&readFileContentInLoopPath, "file_content_in_loop", "", "Path to read the file content in loop from")
 	flag.IntVar(&retryDuration, "retry_time", 180, "Retry time during the loop")
+	flag.BoolVar(&checkInitialContent, "check_initial_content", true, "Check initial content during the loop")
```

Comment:
Rename to `break_on_expected_content`, and make it implement the following semantics:

1.  If `break_on_expected_context` is set, make it implement the existing semantics
2.  If it's not set, just loop until the end of the duration.